### ISSUE TITLE
Friend circles

### ISSUE THREAD

**Issue by elidourado** [devonzuegel/ketchup-club#7]
I think the MVP needs friends/following, but in the long run we may want friend circles. There are times when you may want to accept calls or share location with only a particular subset of friends. In general we can make each piece of data (call availability status, current location, future trips or events) available to the public, only friends, or only particular friends.

**Comment by devonzuegel** [comment]
Yes, I agree. MVP definitely needs a way to have friends, rather than how it is right now where you're automatically friends with everyone who's a user. Even with just a handful of people I find that it's a chilling effect!

I like the idea of the idea of friend circles too. We should explore some designs for how we'd implement that before writing code I think, because there are a lot of different directions it could take.
